src/Arcus.WebApi.Tests.Unit/Security/InMemorySecretProvider.cs

```diff
@@ -3,6 +3,7 @@
 using System.Linq;
```

Comment:
I guess there are some methods that can be removed?

Reply:
Done